Tests/ImmutableXCoreTests/Model/ListAssetsOrderByTests.swift

```diff
@@ -0,0 +1,13 @@
+@testable import ImmutableXCore
+import XCTest
+
+final class ListAssetsOrderByTests: XCTestCase {
+    func testAsApiArgument() throws {
```

Comment:
@dom-murray I've added this test to ensure coverage but also ensure that any updates to `AssetsAPI.OrderBy_listAssets` are caught by this test 👍